packages/odie-client/src/components/message/user-message.tsx

```diff
@@ -9,7 +9,19 @@ import { DirectEscalationLink } from './direct-escalation-link';
 import { GetSupport } from './get-support';
 import { uriTransformer } from './uri-transformer';
 import WasThisHelpfulButtons from './was-this-helpful-buttons';
-import type { Message } from '../../types';
+import type { Chat, Message } from '../../types';
+
+const isWordPressRelatedQuestion = ( message: Message ) => {
+	return ! (
+		message.context?.question_tags?.category === 'Other (everything else)' &&
+		message.context?.question_tags?.inquiry_type === 'other' &&
+		message.context?.question_tags?.product === 'unknown'
+	);
+};
+
+const shouldRenderRatingButtons = ( chat: Chat, message: Message ) => {
+	return isWordPressRelatedQuestion( message ) && chat.provider !== 'zendesk';
+};
```

Comment:
As per our discussion in p1736434881255269, I believe `shouldRenderRatingButtons` should always return true (or `chat.provider !== 'zendesk'`. I don't fully understand the logic here).

Reply:
I am still seeing issues here. The ratings should be shown after the user queries for "human support".